### PR TITLE
#163585039 Database persistence for REST endpoints

### PR DESCRIPTION
**Pull request by perfect090**
**What does this PR do?**
Have the database setup to read and write data into it

**Description of Task to be completed**
Have the following endpoints working
POST /api/v1/auth/signup
POST /api/v1/auth/login
POST /api/v1/offices/:id/register
POST/api/v1/votes

**How should this be manually tested?**
They can be manually tested, using POSTMAN after cloning the repository

**What are the relevant pivotal tracker stories?**
#163317717 #163584910  #163318755 #163318774 #163318836

### CODE REVIEW

File: src/helpers/helper.js [hunk 1]
```diff
@@ -1,19 +1,46 @@
+import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
+import bcryptjs from 'bcryptjs';
 
 dotenv.config();
-
+process.env.DB_SECRET = 'hacking';
```
Comment (JCanaks) [inline src/helpers/helper.js]: Why do we have an assignment to you env variable here? this should be do in your env file

File: src/middlewares/authUser.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   * @returns {object} - JSON response object
+   */
+  static authHeader(req) {
+    if (!req.headers.authorization || !req.headers['x-access-token'] || !req.body.token || req.query.token) {
+      return { error: 'error' };
+    }
+    const token = req.body.token || req.query.token || req.headers['x-access-token'] || req.headers.authorization.split(' ')[1];
+    const payload = Helper.verifyToken(token);
+    if (!payload) {
+      return { error: 'token' };
```
Comment (JCanaks) [inline src/middlewares/authUser.js]: The errors messages are not descriptive

File: src/helpers/helper.js (hunk 1 above)
Comment (perfect090) [reply src/helpers/helper.js]: > @perfect090 Why do we have an assignment to your env variable here? this should be do in your env file

Reply

**Comment by perfect090** [comment]
Screenshots 
screenshot 12

**Comment by perfect090** [comment]
Sreecnshot for api/v1/auth/login
screenshot 13

**Pull request merged**